app/templates/partials/widgets/multiple_choice_widget.html

```diff
@@ -35,12 +40,20 @@
     <div class="field__item js-focusable-box">
       <input {{input|xmlattr}}>
       <label {{label|xmlattr}}>
-        {{option.label.text}}
+          {% if not exclusive or (exclusive and not loop.last) %}
+            {{option.label.text}}
+          {% endif %}
         {% if answer.options[loop.index0].description %}
           <br />
           <span class="label__description label__inner pluto">{{answer.options[loop.index0].description}}</span>
         {% endif %}
+          {% if exclusive and loop.last %}
+            <span class="u-vh">Or,</span> {{option.label.text}}<span class="u-vh">. Selecting this will uncheck all other checkboxes</span>
```

Comment:
Space could probably be within the `<span u-vh`. `<span class="u-vh">Or, </span>{{option.label.text}}`